contracts/test/plugin/setup/plugin-setup.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  IERC165__factory,
+  IPluginSetup__factory,
+  IProtocolVersion__factory,
+} from '../../../typechain';
+import {getInterfaceId} from '@aragon/osx-commons-sdk';
+import {expect} from 'chai';
+import {Contract} from 'ethers';
+
+// TODO
+
+describe.skip('IPluginSetup', async () => {
+  it('has the same interface ID as in OSx v1.0.0', async () => {
+    expect(true).to.equal(false);
+  });
+});
+
+describe.skip('PluginSetup', async () => {
```

Comment:
why are these skipped? there are tests present that should work

Reply:
I just put them there as a placeholder because we don't have a proposal mock yet.

```ts
let contract: Contract; // TODO create ProposalMock
```

Reply:
I'll add placeholders in https://aragonassociation.atlassian.net/browse/OS-943 and unskip the tests afterwards.